README.rst

```diff
@@ -398,6 +398,35 @@ Keys to take special care are the ones needed to configure Kafka and advertised_
      - ``lowest``
      - Decides on what basis the Karapace cluster master is chosen (only relevant in a multi node setup)
 
+Uninstall
+=========
+
+To unistall Karapace from the system you can follow the instructions described below. We would love to hear your reasons for uninstalling though. Feel free to drop us a line letting us know what we can improve!
+
+Installed via Docker
+--------------------
+
+If you installed Karapace via Docker, you would need to first stop and remove the images like described:
+
+First obtain the container IDs related to Karapace, you should have one for the registry itself and another one for the rest interface.
+
+  docker ps | grep karapace
```

Comment:
also code blocks for these? not only for looks but also for easier copying

Reply:
Wasn't code snippets 2 tabs?

Reply:
Got it now